README.md

```diff
@@ -159,9 +157,9 @@ You can submit bug reports using the [GitHub issue tracker](https://github.com/i
 ### Pull requests
 
 Pull requests are welcome.  Before submitting a new pull request, please
-make sure that your code passes both the [pep8](https://www.python.org/dev/peps/pep-0008/) formatting check:
+make sure that your code passes both the [code formatting](https://www.python.org/dev/peps/pep-0008/) formatting check:
```

Comment:
Yeah, I just wanted to drop the term `PEP-8` because the `pep8` tool now has renamed itself to `pycodecheck` because Guido basically told them they were not faithful enough to the standard, so it may be misleading that they name themselves after it. (I have started to really dislike the tool personally, but that's another story…)